schema/module.go

```diff
@@ -0,0 +1,71 @@
+package schema
+
+import (
+	"crypto/md5"
```

Comment:
__[gometalinter]__
* [gas] Use of weak cryptographic primitive,HIGH,HIGH

<!-- Comment by [SideCI](https://sideci.com) -->